default/python/AI/AIFleetMission.py

```diff
@@ -393,6 +393,7 @@ def need_to_pause_movement(self, last_move_target_id: int, new_move_order: Order
         # before proceeding
         return False
 
+    # flake8: noqa: C901
```

Comment:
This disables the warning for the whole file, see https://stackoverflow.com/questions/51127170/flake8-ignore-warnings-for-a-function

Note that I also found that when you put it in the first line of the function, you must write `# noqa: C901` or `# noqa: C901 flake8`. With flake8 before the noqa flake8 will ignore it.

Reply:
Thank you.  Nice catch.